src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -171,17 +171,28 @@ def handler(ex):
                               ResourceType.DATA_STORAGE,
                               'common._error#AzureHttpError')
             if isinstance(ex, t_error) and ex.status_code == 403:
-                message = """
+                if ex.error_code == 'AuthorizationPermissionMismatch':
```

Comment:
What's the message if ex.status_code == 403 and ex.error_code is not in [AuthorizationPermissionMismatch, AuthorizationFailure, AuthenticationFailed]?

Reply:
Throw exception with raw message